Move host benchmarks to a subdirectory for easier distinction

Priya2698 opened the pull request.

naoyam reviewed (approved):
LGTM, but let's rename the directory to `benchmarks/python/host/`.

xwang233 commented:
> I can check if there are any paths in the JSON output that will be changed

Yes, that's the concern. I guess there aren't paths in the JSON output but let's double check.

Dashboard visualization should be fine. We may need to update the CI script that runs benchmark. Currently it runs `test_*.py` at depth 1. I'll have to update the script to search benchmark scripts recursively. This won't be a big problem.

xwang233 commented:
I just verified that there's no change in the path in generated pytest benchmark JSON file. I've also updated the CI script to recursively search `test_*.py` benchmark scripts.

Once the relative import issue is resolved, we're ready to go.

Priya2698 commented:
!build

Priya2698 commented:
> I just verified that there's no change in the path in generated pytest benchmark JSON file. I've also updated the CI script to recursively search `test_*.py` benchmark scripts.
> 
> Once the relative import issue is resolved, we're ready to go.

Thanks, Xiao.
The subdir name is `host`. I also changed `test_many_pointwise_ops.py -> test_many_pointwise_ops_host.py` for consistency. With the subdirectory we may not need `host` in the name, though. I

Priya2698 commented:
!build
